### PR TITLE
Skip tvchaosuk highlighted torrents

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/providers/torrent/html/tvchaosuk.py [hunk 1]
```diff
@@ -135,9 +135,11 @@ def parse(self, data, mode, **kwargs):
             # Skip column headers
             for row in torrent_rows[1:]:
                 try:
+                    # Skip highlighted torrents
+                    if (row['class'] == 'highlight even') or (row['class'] == 'highlight odd'):
```
Comment (medariox) [inline medusa/providers/torrent/html/tvchaosuk.py]: You don't need the brackets here.
Are you sure 'class' will always be there? If you are unsure and want to play it safe, use .get('class') instead.

Comment (duramato) [inline medusa/providers/torrent/html/tvchaosuk.py]: That class will always be there

**Comment by fernandog** [comment]
@duramato @medariox i think we should discard last commit. is safer to check the class name.

**Comment by duramato** [comment]
Checking class name didn't worked at all. From this pr only that commit works

**Comment by medariox** [comment]
I don't know if there is another way, as I don't have access. Checking for 'class' existence looks kind of excessive to me though.

**Comment by medariox** [comment]
@duramato 
Please always add the milestone in the future.

**Pull request merged**